query/src/main/scala/filodb/query/ProtoConverters.scala

```diff
@@ -222,6 +240,40 @@ object ProtoConverters {
     }
   }
 
+  implicit class IndividualQuotaFromProtoConverter(giq: GrpcMultiPartitionQueryService.IndividualQuota) {
+    def fromProto(warnDefaults: Boolean = false): IndividualQuota = {
```

Comment:
It's in inappropriate to add ``warnDefaults`` here. This converter should convert given proto to equivalent case class. Misusing this method will introduce bugs hard to troubleshoot. ``toProto`` and ``fromProto`` should not accept parameters and alter the behavior. . 

Have another one to convert case class to proto and write a spec similar to how it's written for others.

Reply:
need to talk to you about this one I guess. the problem is that it seems like protos don't have default values but scala case classes do. So, suppose you have a proto that does not have a property defined but you cannot NOT to create a corresponding scala object since its properties are not optional. So, either I define proto properties as required or?..  I noticed that most of your protos have the fields defined as optional and I believe you did it for a reason, correct?

Reply:
Correct but we never add parameters to ``toProto`` and ``fromProto``. The goal is to convert whatever the case class has (default or explicitly set values) to proto and back.